apps/mailman/templates/mailman/survey_email_html.py

```diff
@@ -6,63 +6,42 @@ def write_html(email_text, link_html):
 <body style="margin:0; padding: 0;">
 <div align="center">
 <table border="0" cellpadding="0" cellspacing="0" align="center" width="100%" style="font-family: Arial,Helvetica,sans-serif; max-width: 700px;">
-       <tbody><tr bgcolor="#39446f">
-           <td colspan="5" height="40">&nbsp;</td>
-       </tr>
-           <tr bgcolor="#39446f"></tr>
-       <tr bgcolor="#39446f">
-           <td width="20">&nbsp;</td>
-           <td width="20">&nbsp;</td>
-           <td align="center" style="font-size: 29px; color:#FFF; font-weight: normal; letter-spacing: 1px; line-height: 1;
-                           text-shadow: -1px -1px 1px rgba(0, 0, 0, 0.2); font-family: Arial,Helvetica,sans-serif;">
-    <img alt="WorkElections" src="https://s3.amazonaws.com/voteworker/workelections.png">
-        <br><br>
-              WorkElections.com Request for Information
-           </td>
-           <td width="20">&nbsp;</td>
-           <td width="20">&nbsp;</td>
-       </tr>
-       <tr bgcolor="#39446f">
-           <td colspan="5" height="40">&nbsp;</td>
-       </tr>   
-       <tr>
-           <td height="20" colspan="5">&nbsp;</td>
-       </tr>
-    <tr>
+    <tbody><tr>
+       <td align="center" style="padding-left:30px; padding-right:30px"><img width="200" alt="WorkElections" src="https://s3.amazonaws.com/voteworker/workelections.png"></td>
+       <td align="center" style="padding-left:30px; padding-right:30px"><img width="240" alt="Fair Elections Center" src="https://www.workelections.com/graphics/meta/FairElectionsCenterLogo.png"></td>
```

Comment:
These are the image urls that should be served from CDN.

Reply:
Would Google Drive be a good option for this?

Reply:
Looks like they used to support this, but it was turned down: https://stackoverflow.com/questions/10311092/displaying-files-e-g-images-stored-in-google-drive-on-a-website